features/FEATURE_PSA/TARGET_TFM/src/os_wrapper_cmsis_rtos_v2.c

```diff
@@ -168,3 +202,41 @@ void os_wrapper_thread_exit(void)
 {
     osThreadExit();
 }
+
+uint32_t os_wrapper_thread_delete(void *handle)
+{
+    osStatus_t status = osOK;
+
+    if (!handle) {
+        return OS_WRAPPER_ERROR;
+    }
+
+    osRtxThread_t *thread = (osRtxThread_t *)handle;
+    void *st_mem = thread->stack_mem;
+
+    status = osThreadTerminate((osThreadId_t)handle);
+    if (status != osOK) {
+        return OS_WRAPPER_ERROR;
+    }
+
+    free(st_mem);
+    free(handle);
```

Comment:
Call `delete` on the Mbed OS C++ Thread object instead. Use the address of the thread object as your `void *handle`.

Reply:
Fixed in a later commit.